recipes/xsum/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "xsum" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 21ff547dbb64e8ddefd376f2889349c7a94fce69e7fc417fc661068b298473b1
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python 
+    - pip
+    - pybind11
+  run:
+    - python
+    - pybind11
+    - libcxx
+    - numpy
```

Comment:
Are you linking against numpy or just using it at runtime?

Reply:
> Are you linking against numpy or just using it at runtime?

@scopatz I am using `pybind11/numpy.h` header file in my package during the build process. I am also using `numpy` during the runtime.

Reply:
Please see the instructions here: https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Reply:
@scopatz  thanks. I have read this instruction before. Since I do not link against `numpy` directly I do not think it concerns me. I am just using it during runtime and including the 'pybind11/numpy.h' and pybind11 has been built before.

Reply:
@scopatz If it is advised I add a version I can put a minimum `numpy` version there.

Reply:
I am confused, you say you are using it during the build process, but it is not a host dep?

Reply:
@scopatz Sorry for my mistake. I am just using `pybind11/numpy.h` header file and no linking with `numpy`.

Reply:
But just at runtime? Not during the build?

Reply:
@scopatz yes

Reply:
@scopatz I am not exactly sure about the `pybind11/numpy.h` header file. It seems it is not linking with `numpy`, but I can test it to make sure.

Reply:
@scopatz Actually it seems in Pybind11 you do not link against `numpy`.